entity_gym/entity_gym/environment.py

```diff
@@ -171,15 +172,13 @@ class Environment(ABC):
     This should be replaced by a more general multi-agent environment interface in the future.
     """
 
```

Comment:
I'm completely in favour of moving this to an instance method rather than a class method, see #23 for additional possible issues that can be solved with this choice.